src/data/nat/prime.lean

```diff
@@ -1019,7 +1042,35 @@ begin
   { simp [(coprime_zero_left _).mp hab] },
   rcases b.eq_zero_or_pos with rfl | hb,
   { simp [(coprime_zero_right _).mp hab] },
-  rw [mem_factors_mul_of_pos ha hb p, list.mem_union]
+  rw [mem_factors_mul_of_pos ha hb p, list.mem_union],
 end
 
+/-- If `x` is a prime factor of `a` then `x` is also a prime factor of `a * b` for any `b > 0` -/
+lemma mem_factors_mul_left {x a b : ℕ} (hax : x ∈ a.factors) (hb : 0 < b) : x ∈ (a*b).factors :=
```

Comment:
Can you rename `x` to `p` in this and the following lemmas?

Reply:
It might be useful to have a lemma of the form 
`lemma le_factors_count_mul (hb : 0 < b) : list.count p b.factors \le list.count p (a * b).factors`

Reply:
That's a good idea, I've added `lemma le_factors_count_mul {p a b : ℕ} (ha : 0 < a) : list.count p b.factors ≤ list.count p (a * b).factors`